### ISSUE TITLE
Bug that fills the console with messages

### ISSUE THREAD

**Issue by Toketo1442** [jdevs-mc/ChristmasGifts#4]
```
[05:56:38 WARN]: [ChristmasGifts] Task #6565 for ChristmasGifts v1.1.2 generated an exception
java.lang.IllegalArgumentException: bound must be positive
at java.base/java.util.Random.nextInt(Random.java:557) ~[?:?]
at java.base/java.util.concurrent.ThreadLocalRandom.nextInt(ThreadLocalRandom.java:468) ~[?:?]
at ChristmasGifts.jar/dev.jdevs.JGifts.events.SpawnGifts$3$1.run(SpawnGifts.java:95) ~[ChristmasGifts.jar:?]
at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:86) ~[purpur-1.21.jar:1.21-2284-4e55e26]
at org.bukkit.craftbukkit.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:475) ~[purpur-1.21.jar:1.21-2284-4e55e26]
at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1762) ~[purpur-1.21.jar:1.21-2284-4e55e26]
at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:509) ~[purpur-1.21.jar:1.21-2284-4e55e26]
at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1634) ~[purpur-1.21.jar:1.21-2284-4e55e26]
at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1334) ~[purpur-1.21.jar:1.21-2284-4e55e26]
at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:333) ~[purpur-1.21.jar:1.21-2284-4e55e26]
at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
```

---------------------------------------------------------------------------------------

This message appears all the time in the console, it fills the log causing the server to weigh 500mb for having been on for approximately 5/6 hours.

Every second about 20 messages appear, each tick a message appears, to be more precise.

**Comment by mitriyf** [comment]
Can I get your launch.yml to identify the problem?

**Comment by mitriyf** [comment]
Not required, I have identified the problem. Your server was low online, sorry for the misunderstanding. Today I will try to finish optimizing the code, as well as fix this problem (The problem has already been fixed, the code is in the process)
Sorry.
May I ask why you use gift giving every second?

**Comment by mitriyf** [comment]
This was resolved in the v1.1.3-release update.
Sorry.
